titan/partnering.py

```diff
@@ -141,6 +141,7 @@ def get_mean_rel_duration(params: "parse_params.ObjMap"):
             mean_rel_duration[bond] = params.partnership.duration[
                 bond
             ].distribution.mean
+        assert mean_rel_duration[bond] > 0, "All bonds must have a positive duration!"
```

Comment:
is this a common problem? is there a way to catch it earlier?

Reply:
It should be quite uncommon, and I don't think there's an easy way to check earlier (since figuring out the mean of the bins is a bit complicated). It's just to stop no partnerships from forming if your durations are impossible.

Reply:
I guess this is pretty early in the process actually, since it's just at population creation, right?